### ISSUE TITLE
Save global position and zoom only once

### ISSUE THREAD

**Issue by devsts** [umlet/umlet#549]
Hi, I have added my uxf-files to a Git repository. It is impractical for comparing changes that the coordinates of all elements are changed when the whole diagram is moved or zoomed.

Can you implement in the next version that global position an zoom is only saved once and the coordinates of all elements are relative to them?

**Comment by afdia** [comment]
this is a nice idea but unfortunately a bit more complicated than it seems
duplicate of #527
